test/converter.js

```diff
@@ -146,11 +146,93 @@ test('boolean attributes', async function (test) {
   await testAttrbuteType(test, 'boolean', 'Boolean');
 });
 
+// ISSUE 34 CUSTOM TEST CASE 1.3
+test('null | string attributes', async function (test) {
+  await testAttrbuteType(test, ['null', 'string'], 'String');
```

Comment:
I would have expected the result type to be some form of an optional `String`. How does the type of 'string' differ from ['null', 'string'] ?

Reply:
Well, GraphQL will return nullable types by default. I've only handled the processing of arrays so far. But following your logic, it seems that type 'string' should return `String!`. And this should be the case for 'number', 'integer', and 'boolean'. I'm looking into this right now.

Reply:
Just used the tool on the following JSON file:

`{
  "$id": "DummyData",
  "type": "object",
  "properties": {
    "attribute": {
      "type": "string"
    }
  }
}`

Output looked like this:

`
type DummyData {
  attribute: String  // Notice this type is nullable by default
}

input DummyDataIn {
  attribute: String
}

type Mutation {
  DummyData(input: DummyDataIn): String
}
`

At this point, I think my previous comment was on the money. To answer your question, there isn't any difference between the return type from `['null', 'string']` and `'string`.

Reply:
Your interpretation sounds right to me. There is a way to mark an attribute as required in JSON schemas, but it is different than just using 'string'. Good catch!